auth/src/main/java/foundation/algorand/auth/crypto/KeyPairs.kt

```diff
@@ -33,17 +31,16 @@ class KeyPairs {
             }
         }
         fun getKeyPair(): KeyPair {
-            return getKeyPair(Account().toMnemonic())
+            val mnemonicCode: MnemonicCode = MnemonicCode(WordCount.COUNT_24)
+            return getKeyPair(mnemonicCode.joinToString(" "))
         }
         fun getKeyPair(mnemonic: String): KeyPair {
             Log.d(TAG, "getKeyPair(******)")
             val generator = KeyPairGenerator.getInstance(KEY_ALGO)
-            val seed = Mnemonic.toKey(mnemonic)
-            val fixedRandom = FixedSecureRandom(seed)
+            val entropy = MnemonicCode(mnemonic.toCharArray()).toEntropy()
```

Comment:
I wish we could switch to BIP39 but it is out of scope. The demo application would also need to be refactored since it's using the Java SDK's `Account`. We just need `FixedSecureRandom` and `KeyPairs` to be in the `demo` namespace

- revert BIP39
- move `foundation.algorand.auth.crypto.FixedSecureRandom` -> `foundation.algorand.demo.crypto.FixedSecureRandom`
- move `foundation.algorand.auth.crypto.KeyPairs` -> `foundation.algorand.demo.crypto.KeyPairs`
- update imports in the `foundation.algorand.demo`